src/colmap/controllers/incremental_pipeline_test.cc

```diff
@@ -203,7 +203,7 @@ TEST(IncrementalPipeline, PriorBasedSfMNoNoise) {
   Reconstruction gt_reconstruction;
   SyntheticDatasetOptions synthetic_dataset_options;
   synthetic_dataset_options.num_cameras = 2;
-  synthetic_dataset_options.num_images = 7;
+  synthetic_dataset_options.num_images = 10;
```

Comment:
We use unordered_map's in various places, which have an undefined iteration order, which differs between standard library implementations, in particular MSVC vs libc++/libstdc++. Using more images, makes the alignment in the prior mapper more stable.